### PR TITLE
Fix jitter on Windows

### PR DESCRIPTION
**Pull request by jbgalet**
On Windows with Python38 the Jitter fails to compile mostly because of the native extension filenames `.cp38-win_amd64.lib`.
Therefore:
- during setup, some libs are misplaced
- bad extension in `jitcore_cc_base.py` (`sysconfig.get_config_var('EXT_SUFFIX')` is `.pyd` but we need `.lib` here)
- use hardcoded `python27.lib`

### CODE REVIEW

**Comment by serpilliere** [comment]
Hi @jbgalet !
If windows with python38 is now supported, can you upgrade the `.appveyor.yml` file in this PR to add the regression test for this platform?

**Comment by serpilliere** [comment]
If you are not familiar with appveyor yml, don't hesitate to tell me, I will PR your PR with the fix!

**Comment by serpilliere** [comment]
Thanks a lot @jbgalet!

Don't worry for the travis bugs, it's a llvm bug we are facing for some times.
Just a last request if you are ok: Can you merge your first and second commit ?
(the second one is just a fixup)

**Comment by jbgalet** [comment]
Should be good, note that I had to update the llvmlite requirement to 0.31.0 to make it work, but it breaks travis (since llvmlite 0.31 requires llvm 7).
I have a .travis.yml that almost works here: https://github.com/jbgalet/miasm/blob/test-travis/.travis.yml
I can merge it as needed.

**Comment by serpilliere** [comment]
Thank you @jbgalet !
If you are ok, you can PR the fix for the travis!

**Comment by serpilliere** [comment]
@jbgalet , with your travis file, did you manage to make llvm tests pass again on travis ?

File: setup.py [hunk 1]
```diff
@@ -248,7 +249,8 @@ def buil_all():
         for lib in libs:
             filename = os.path.basename(lib)
             dst = os.path.join(build_base, lib_dirname, "miasm", "jitter")
-            if filename not in ["VmMngr.lib", "Jitgcc.lib", "Jitllvm.lib"]:
+            # Windows built libraries may have a name like VmMngr.cp38-win_amd64.lib
+            if not any([fnmatch.fnmatch(filename, pattern) for pattern in ["VmMngr.*lib", "Jitgcc.*lib", "Jitllvm.*lib"]]):
```
Comment (p-l-) [inline setup.py:253]: I know it's too late, but you don't need to build a list in such situations; this works:
```suggestion
            if not any(fnmatch.fnmatch(filename, pattern) for pattern in ["VmMngr.*lib", "Jitgcc.*lib", "Jitllvm.*lib"]):
```

**Comment by jbgalet** [comment]
> @jbgalet , with your travis file, did you manage to make llvm tests pass again on travis ?

Not yet. I've talked to @commial and it seems there are some issues related to #1153

**Comment by serpilliere** [comment]
Oh ok!
I thought you have found it was fixed in the last version of llvm.
Noproblemo.

**Pull request merged**